Explorer/Assets/DCL/Passport/Modules/UserDetailedInfo_PassportModuleController.cs

```diff
@@ -0,0 +1,664 @@
+
+            LayoutRebuilder.ForceRebuildLayoutImmediate(view.MainContainer);
+        }
+
+        private void CreateNewLink(string title, string url)
+        {
+            AddLink(Guid.NewGuid().ToString(), title, url, true);
+            LayoutRebuilder.ForceRebuildLayoutImmediate(view.MainContainer);
+        }
+
+        private void RemoveLink(Link_PassportFieldView linkToRemove)
+        {
+            var indexToRemove = 0;
+            foreach (var link in instantiatedLinksForEdition)
+            {
+                if (link.Id == linkToRemove.Id)
+                    break;
+
+                indexToRemove++;
+            }
+
+            if (indexToRemove >= instantiatedLinksForEdition.Count)
+                return;
+
+            linksPoolForEdition.Release(linkToRemove);
+            instantiatedLinksForEdition.RemoveAt(indexToRemove);
+            SetNewLinkButtonActive(true);
+        }
+
+        private void SetNewLinkButtonActive(bool isActive)
+        {
+            view.AddNewLinkButton.gameObject.SetActive(isActive);
+
+            if (isActive)
+                view.AddNewLinkButton.transform.SetAsLastSibling();
+        }
+
+        private void SaveLinksSection()
+        {
+            saveLinksCts = saveLinksCts.SafeRestart();
+            SaveLinksAsync(saveLinksCts.Token).Forget();
+            return;
+
+            async UniTaskVoid SaveLinksAsync(CancellationToken ct)
+            {
+                SetLinksSectionAsSavingStatus(true);
+
+                if (currentProfile.Links == null)
+                    currentProfile.Links = new List<LinkJsonDto>();
+                else
+                    currentProfile.Links.Clear();
+
+                List<LinkJsonDto> linksToSave = new ();
+                foreach (var link in instantiatedLinksForEdition)
+                {
+                    currentProfile.Links.Add(new LinkJsonDto
+                    {
+                        title = link.Title.text,
+                        url = link.Url,
+                    });
+                }
+
+                await UpdateProfileAsync(ct);
+                ClearAllLinks();
+                LoadLinks();
+                SetLinksSectionAsEditionMode(false);
+                LayoutRebuilder.ForceRebuildLayoutImmediate(view.MainContainer);
+            }
+        }
+
+        private void SetLinksSectionAsSavingStatus(bool isSaving)
+        {
+            view.SaveLinksButtonLoading.SetActive(isSaving);
+            view.CancelLinksButton.gameObject.SetActive(!isSaving);
+            view.SaveLinksButton.gameObject.SetActive(!isSaving);
+            view.SaveInfoButton.interactable = !isSaving;
+
+            foreach (var buttonToDisable in view.ButtonsToDisableWhileSaving)
+                buttonToDisable.interactable = !isSaving;
+
+            foreach (var link in instantiatedLinks)
+                link.SetAsInteractable(!isSaving);
+        }
+
+        private async Task UpdateProfileAsync(CancellationToken ct)
```

Comment:
It should be a `UniTask` for sure

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/1277/commits/387d7cf255586bb55d2b58e802867f3e8d31c408